src/ui/ui_manager.py

```diff
@@ -271,10 +271,67 @@ def _keep_item(self, item_finder: ItemFinder, img: np.ndarray) -> bool:
         wait(0.2, 0.3)
         _, w, _ = img.shape
         img = img[:, (w//2):,:]
-        item_list = item_finder.search(img)
-        item_list = [x for x in item_list if "potion" not in x.name and not \
-            (self._config.items[x.name] == 3 and not self._template_finder.search("ETHEREAL", img).valid)]
-        return len(item_list) > 0
+        original_list = item_finder.search(img)
+        filtered_list = []
+        for x in original_list:
+            if "potion" in x.name: continue
+            include_props = self._config.items[x.name].include
+            exclude_props = self._config.items[x.name].exclude
+            if not (include_props or exclude_props):
+                Logger.debug(f"{x.name}: Stashing")
+                filtered_list.append(x)
+                continue
+            include = True
+            include_bool_type = self._config.items[x.name].include_type
```

Comment:
I find the name confusing. `include_bool_type` suggests that this variable is a bool while it really is a str ["OR", "AND"]